interfaces/evse_board_support.yaml

```diff
@@ -14,31 +15,31 @@ cmds:
       country_code:
         description: A two-letter country code in ISO 3166-1 alpha-2 format
         type: string
-      rcd_enabled:
-        description: 'true: enable RCD, false: disable RCD'
-        type: boolean
   get_hw_capabilities:
-    description: Get Hardware capability/limits
+    description: >-
+      Get Hardware capability/limits. For AC these are the limits of the power path (e.g. relais etc). 
+      For DC, these are the limits for the AC input of the ACDC converter stack, i.e. the complete AC input.
+      Note that DC output limits are reported by the DC power supply itself.
     result:
       description: Hardware capability/limits
       type: object
-      $ref: /board_support#/HardwareCapabilities
+      $ref: /evse_board_support#/HardwareCapabilities
   enable:
```

Comment:
Is it planned to introduce a cmd to enable/disable the CP (CP state E). Some hardware driver will be able to configure this state (e.g. Charge Control C hardware).

Reply:
The BSP has an enable(true/false) command that normally switches to state F to indicate EVSE not available. Yeti also has the capability to switch CP to high impedance. There are no valid procedures defined in IEC for that afaik, but we use it e.g. during virtual replugging during testing (along with PP replugging). This is done by evse_replug command. The implementation of how the replugging is done is left to the BSP (could e.g. be 0V on CP and a short interruption on PP or similar). Currently this is used in the "AC with SoC" mode which is start with DC until SoC is received, replug, charge AC PWM. Is there any other situation where this can be useful?

Reply:
reinit also allows F instead of E, and most hw implementations cannot switch to E so I think F is the better choice for reinit